### PR TITLE
Array completion

### PR DESCRIPTION
**Pull request by mrollinsiv**
Complete logic to handle running differences of non-literal value arrays.
Bump version.

### CODE REVIEW

**Review by gabrielpascua — APPROVED** [review]
Looks good

File: test/arrays.js [hunk 1]
```diff
@@ -194,6 +194,62 @@ describe('Array Diff', function() {
     ];
     assert.deepEqual(diff(parent, theirs, mine), expected);
   });
+
```
Comment (shawnpwalsh) [inline test/arrays.js:197]: Possible additional tests (not sure whether they'd return as expected, could be stubbed out for future if we aren't concerned now):

- Swap order of objects in array
- What if array doesn't contain objects (or some objects and some primitive values)?

Comment (mrollinsiv) [reply test/arrays.js:197]: I'm not concerned right now, but definite idea for the future.  I'll create an issue for it.

**Review by shawnpwalsh — APPROVED** [review]

File: test/arrays.js (hunk 1 above)
Comment (mrollinsiv) [reply test/arrays.js:197]: Created issue #11

**Pull request merged**